src/main/java/org/humancellatlas/ingest/errors/ErrorType.java

```diff
@@ -1,4 +1,4 @@
-package org.humancellatlas.ingest.submission;
+package org.humancellatlas.ingest.errors;
```

Comment:
Just a very minor comment, this could also be just `submissionerror` since the error is in the context of submissions only. It's also consistent with current package submissionmanifest. I'm also not sure what the best practice here, so feel free not to follow this. :)

Reply:
Do you mean the package could be `org.humancellatlas.ingest` and the class names stay as `SubmissionError` and `IngestError`?